src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -310,30 +311,37 @@ private async Task<Solution> EncapsulateFieldsInCurrentProcessAsync(Document doc
                 RenameInComments: false,
                 RenameFile: false);
 
-            var initialLocations = await Renamer.FindRenameLocationsAsync(
+            var initialLocations = await Renamer.FindLightweightRenameLocationsAsync(
                 solution, field, options, fallbackOptions, cancellationToken).ConfigureAwait(false);
 
-            var resolution = await initialLocations.Filter(filter).ResolveConflictsAsync(
-                finalName, nonConflictSymbols: null, cancellationToken).ConfigureAwait(false);
+            var resolution = await ConflictResolver.ResolveLightweightConflictsAsync(
+                initialLocations.Filter(filter), finalName, nonConflictSymbols: null, cancellationToken).ConfigureAwait(false);
```

Comment:
"encapsulate field" and "use auto prop" are examples of two other features that engage with the rename engine in a two-phase fashoin (getting the locations to rename, adapting them, and then asking for the rename to be done).

Previously, this would force all the renamed symbols to be marshalled back to the host, now taht no longer happens.